src/courses/course1/page1.js

```diff
@@ -20,5 +20,6 @@ module.exports = {
       check: (inputs) => inputs.id === 'red',
     },
   ],
+  previous: null,
```

Comment:
Perhaps we'll leave the previous/next checks to the frontend, as according to the API these previous/next pages are not mandatory, and there's no previous page for the starting page, as well as there's no next page for the last page

Reply:
ok